shared/control/parallel_dd.control

```diff
@@ -10,12 +10,16 @@ the files system.
 """
 
 # YOU NEED TO SPECIFY A FILESYSTEM
-if os.path.exists("/dev/sdb"):
-    fs = job.filesystem('/dev/sdb', job.tmpdir)
-elif os.path.exists("/dev/hdb"):
-    fs = job.filesystem("/dev/hdb", job.tmpdir)
-else:
-    fs = job.filesystem("/dev/vdb", job.tmpdir)
+path = glob.glob("/dev/[hsv]d?")
+path.sort()
+fs = path[-1]
+platform = subprocess.check_output("uname -m", shell=True)
```

Comment:
I think better to 
if "ppc" in platform:
      do something
else:
     keep old logical.